README.md

```diff
@@ -90,142 +95,74 @@ After you're finished please remove all the comments and instructions!
   </ul>
 </details>
 
-<!-- Features -->
-
-### Key Features <a name="key-features"></a>
-
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://www.loom.com/share/fd496c8b503549bb900b1665a06e6663)
 
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly deploy your project and add a live demo link in the readme. If your want you can use this **https://render.com/** to deploy it online and you use this **[article](https://microverse.zendesk.com/hc/en-us/articles/10903290796179)** to help you how to proceed.